### ISSUE TITLE
Version 0.8.7 is not compatible with 0.8.6

### ISSUE THREAD

**Issue by mateusduboli** [akka/reactive-kafka#142]
The version `0.8.7` is using the new package structure, namely `akka.kafka`, were as version `0.8.6` uses the old package structure `com.softwaremill.react.kafka`, this is wrong according to semver specifications, and will generate lots of issues during upgrade.

**Comment by patriknw** [comment]
That looks very wrong. Was 0.8.7 released from wrong branch, @kciesielski ?
The `0.8` branch still looks correct.
https://github.com/akka/reactive-kafka/tree/0.8/core/src/main/scala

**Comment by kciesielski** [comment]
@mateusduboli @patriknw I just downloaded and unzipped 0.8.7 from Maven Central (https://repo1.maven.org/maven2/com/softwaremill/reactivekafka/reactive-kafka-core_2.11/0.8.7/). Looks like it's all fine (`com.softwaremill.react.kafka`).

**Comment by kciesielski** [comment]
Ok I got it. The GH release is on the master branch, so if you're downloading zip with sources from GitHub, you're gonna get code for 0.11. I'll fix this.

**Comment by patriknw** [comment]
Ok, i think i looked at the 0.8.7 github tag and the content was wrong, but perhaps it was me that was wrong

**Comment by kciesielski** [comment]
The tag was incorrect and I fixed it. The jar, however, was released from
the correct branch.
On Apr 29, 2016 19:16, "Patrik Nordwall" <notifications@github.com> wrote:

Ok, i think i looked at the 0.8.7 github tag and the content was wrong, but
perhaps it was me that was wrong

—
You are receiving this because you modified the open/close state.

Reply to this email directly or view it on GitHub
<https://github.com/akka/reactive-kafka/issues/142#issuecomment-215819308>
